plugins/views/frontend/public/stylesheets/main.css

```diff
@@ -6,3 +6,17 @@
 .routename-views-home .views-date-picker-container {
   margin-bottom: 32px;
 }
+
+.bu-is-6 .viewsDashboardWidget .cly-vue-metric-cards .cly-vue-metric-card {
```

Comment:
A couple of things here:

-First I'm not sure if it is a good idea to limit the rule to bu-is-6. Any specific reasons for that?
-camelCase / kebab-case confusion here as well. 
-.cly-vue-metric-cards and .cly-vue-metric-card being overridden is a small violation but I guess you had no choice, right?

All these apply to following rules